main.rb

```diff
@@ -1,43 +1,7 @@
+# main.rb
 require_relative 'app'
+require_relative 'user_interface'
 
-class Main
```

Comment:
- [ ] Following the project requirement you are to make sure that the role of main.rb is only to start their code and ask the user for the initial choice of options (or to quit). Please kindly follow this requrement :+1: :sparkles: